src/main/java/uk/gov/hmcts/et/common/model/ccd/items/ListTypeItem.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.et.common.model.ccd.items;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@NoArgsConstructor
+public class ListTypeItem<T> extends ArrayList<GenericTypeItem<T>> {
+    public static <T> ListTypeItem<T> from(T value) {
+        ListTypeItem<T> typeItem = new ListTypeItem<>();
+        typeItem.add(GenericTypeItem.from(value));
+        return typeItem;
+    }
+
+    public static <T> ListTypeItem<T> from(T value, String id) {
+        ListTypeItem<T> typeItem = new ListTypeItem<>();
+        typeItem.add(GenericTypeItem.from(id, value));
+        return typeItem;
+    }
+
+    @SafeVarargs public static <T> ListTypeItem<T> from(T...values) {
```

Comment:
should this annotation be put in the line before?

Reply:
Good question, annotations usually go above, but IntelliJ inserted this for me on the same line, not sure if its because it relates to the params only